### ISSUE TITLE
Poor alt text on index page

### ISSUE THREAD

**Issue by jsoref** [robusta-dev/robusta#229]
**Describe the bug**

**To Reproduce**
1. Turn off loading images in your web browser
2. Go to https://docs.robusta.dev/master/

**Expected behavior**
Alt text should be meaningful.

If the logo is the logo for `Robusta`, then the word `Robusta` should be in the alt text.

**Screenshots**
<img width="244" alt="image" src="https://user-images.githubusercontent.com/2119212/154860183-937ada34-3fad-4879-814b-4e1294872382.png">

**Desktop (please complete the following information):**
 - OS: macOS Big Sur
 - Browser Chrome
 - Version 98

**Additional context**

**Comment by aantn** [comment]
Thank you for reporting!

**Comment by kranurag7** [comment]
@aantn Should I fix this with `robusta-light` and `robusta-dark` in case of `Light Logo` and `Dark Logo` in the dark mode.

**Comment by jsoref** [comment]
I question the value of telling a blind user that the logo is light or dark. I think it's more important to say "Robusta Logo" if that's what it is. If a blind user is curious, they can inspect the file name, which I imagine would tell them the fact that it's light or dark.

Note that blind users don't naturally get told the color of the rest of a web page (neither the foreground nor the background). They can inspect if they're curious, but what they generally want is a description of the meaning of the content, not the presentation of the content.

**Comment by aantn** [comment]
Writing "Robusta Logo" sounds good to me too.

**Comment by kranurag7** [comment]
Great suggestion @jsoref Let's go with `Robusta Logo`
